scripts/clean-db-dump.sql

```diff
@@ -112,6 +112,36 @@ WHERE declaration IS NOT NULL
   AND declaration::varchar != 'null'
   AND declaration::varchar != '{}';
 
+-- Create some fake draft and closed briefs using eligible suppliers
+INSERT INTO brief_responses (
+      data,
+      brief_id,
+      supplier_id,
+      created_at,
+      submitted_at
+)
+SELECT
+    '{"essentialRequirements": [{"evidence": "They have the correct number of hooves."}, {"evidence": "So shiny..."}, {"evidence": "All at least 50"}], "niceToHaveRequirements": [{"yesNo": false}, {"yesNo": true, "evidence": "No jump is too high."}, {"yesNo": true, "evidence": "NEIGH"}], "availability": "09/09/17", "essentialRequirementsMet": true, "respondToEmailAddress": "example-email@gov.uk"}'::json,
```

Comment:
The number of essential requirements and n2h requirements may not be equal to the number the brief is looking for. Can you think of any implications of this? 

For example, I think... if someone logged in as the buyer of a random brief and tried to download the responses, it might fail. 

Trying to figure out if this is acceptable or not (it might be a fair bit harder to create the correct number of pieces of evidence)

Reply:
I think that provisioning for this would take a lot of work.
```
digitalmarketplace=# select len, count(len) from (select json_array_length((data->>'essentialRequirements')::json) as len from briefs) as len_table group by len;
 len | count 
-----+-------
     |     0
   8 |    82
  16 |    16
  15 |    22
   4 |  3172
  20 |    13
   1 |    40
  13 |    41
   5 |   121
  11 |    49
   3 |   135
  14 |    26
  17 |    18
  19 |    20
  12 |    39
  10 |    96
  18 |    21
   9 |   133
   6 |   117
   2 |   126
   7 |   112
```

The majority of briefs have 4 essential requirements. I could try and find out if there's some sql to make them match but I can't think of any off the top of my head.

Reply:
Ah I'd need to add another essential requirement to the test data for a start.